cli/template/extras/src/env/with-auth-db-planetscale.js

```diff
@@ -0,0 +1,60 @@
+import { createEnv } from "@t3-oss/env-nextjs";
+import { z } from "zod";
+
+export const env = createEnv({
+  /**
+   * Specify your server-side environment variables schema here. This way you can ensure the app
+   * isn't built with invalid env vars.
+   */
+  server: {
+    DATABASE_URL: z.string().url(),
```

Comment:
this looks like a planetscale file, shouldn't that have the same schema as the `with-db-planetscale`, i.e. the check for default URL?

Reply:
yea